src/legacy/core_plugins/kbn_vislib_vis_types/public/goal.js

```diff
@@ -38,7 +38,7 @@ export default function GoalVisType(Private) {
         addTooltip: true,
         addLegend: false,
         isDisplayWarning: false,
-        type: 'gauge',
+        type: 'goal',
```

Comment:
this is not needed, pls revert this change

Reply:
this was changed as a last resort.